plugins/slack_app/app/models/slack_message.rb

```diff
@@ -0,0 +1,116 @@
+class SlackMessage
+  def initialize(deploy)
+    @deploy = deploy
+  end
+
+  def deliver
+    url = DeployResponseUrl.find_by(deploy_id: @deploy.id).try(:response_url)
```

Comment:
maybe leave that as comment in the file

On Sun, Jul 17, 2016 at 3:59 PM, Ben Straub <notifications@github.com>
wrote:

> In plugins/slack_app/app/models/slack_message.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r71091589>:
>
> > @@ -0,0 +1,116 @@
> > +class SlackMessage
> > +  def initialize(deploy)
> > +    @deploy = deploy
> > +  end
> > +
> > +  def deliver
> > +    url = DeployResponseUrl.find_by(deploy_id: @deploy.id).try(:response_url)
>
> Only deploys triggered by the slash-command will have an entry in this
> table, so we should fail softly if we don't have a place to send the
> message. This seemed like a reasonable way to deal with that situation.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/d008e855296f051e83d429a560e4e921b94fd917#r71091589>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ1ay8CeRuzYQW_fFveVomSVJy9Q7ks5qWrPugaJpZM4JIRHk>
> .
>